### PR TITLE
overview: Add the data flow diagram for data flowing out of Repo to Overview

### PR DESCRIPTION
**Pull request by tzhelyazkova**
This diagrams is technical and is a good overview of how data flows out of Repo and into Client and into other systems such as Query Service.
The part about babel, where no Repo data is involved, has been removed.
The diagram has been adjusted to fit into the page in a way that the reader can see the text in the boxes.
The colors have been preserved as the ones used in Conceptboard because the things described in this diagram are mostly not mentioned in other diagrams.

p.s. I can read the diagram well on my 27 inch monitor. It's very hard to read it on the 13 inch laptop screen. Not sure about anything in between. The Vuepress zoom plugin does not help either. :mag:

### CODE REVIEW

File: systems/overview/03-Context_and_Scope.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Context and Scope
```
Comment (Silvan-WMDE) [inline systems/overview/03-Context_and_Scope.md:1]:
```suggestion
# Context and Scope
```

File: systems/overview/03-Context_and_Scope.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
+![Legend](../../diagrams/legend.drawio.svg)
+
 ### Business Context
```
Comment (Silvan-WMDE) [inline systems/overview/03-Context_and_Scope.md:5]:
```suggestion
## Business Context
```

File: systems/overview/03-Context_and_Scope.md [hunk 3]
```diff
@@ -17,3 +19,5 @@
 ### Technical Context
```
Comment (Silvan-WMDE) [inline systems/overview/03-Context_and_Scope.md:19]:
```suggestion
## Technical Context
```

**Review by Silvan-WMDE — APPROVED** [review]
I think it's a pity the diagram has to be squashed in order to fit on the page, but I don't have a better solution. What would be needed is some way of zooming _into_ the image to show only a specific part (like Client or Repo) rather than zooming out and display it entirely in fullscreen.


I also added a fix for the broken sub-sections:
image

**Review by jakobw — APPROVED** [review]

**Pull request merged**